vJoySerialFeeder/SerialProtocols/CrsfReader.cs

```diff
@@ -84,7 +84,7 @@ public override int ReadChannels()
 				Buffer.Slide(1);
 				return 0;
 			}
-
+			Buffer.FrameLength = len + 2;
```

Comment:
Isn't the full length `len + 2 + 1`  or `len + 2`?
It seems it's the second. In that case the `Slide()` should be also `len+2`. It works now because any larger number will work but it will be less confusing if we set the correct value. Slide() removes `n` bytes for the buffer so to speak. The correct thing is to remove exactly `Buffer.FrameLength` bytes. It can be written as `Buffer.Slide(Buffer.FrameLength)`

Reply:
You're right. That's a leftover from my initial implementation where my count of how many bytes there should be was one too high.